packages/app/features/settings/EditProfileScreen.tsx

```diff
@@ -22,12 +22,20 @@ export function EditProfileScreen({ route, navigation }: Props) {
     [navigation]
   );
 
+  const handleGoBack = useCallback(() => {
+    // We'll always go back to the profile screen of the user being edited
```

Comment:
This appears to work and I don't think there's anywhere else that we link to _EditProfileScreen_ from.

Setting `backBehavior="history"` on the ActivityNavigator seems to achieve the same thing. I wonder if we might want to do that to avoid special casing here?